localstack/testing/pytest/snapshot.py

```diff
@@ -103,16 +98,8 @@ def pytest_runtest_call(item: Item) -> None:
         sm._assert_all(verify, paths)
 
 
-@pytest.fixture(name="region", scope="session")
-def fixture_region(aws_client):
-    if is_aws() or is_api_enabled("sts"):
-        return aws_client.sts.meta.region_name
-    else:
-        return TEST_AWS_REGION_NAME
```

Comment:
This is moved to `localstack.testing.pytest.fixtures` alongside its companion fixtures